apidoc/Titanium/UI/ScrollableView.yml

```diff
@@ -93,6 +93,22 @@ methods:
         summary: An integer index or <Titanium.UI.View> object to set as the current page.
         type: [Number, Titanium.UI.View]
 
+  - name: setIndicatorImageForPage
+    summary: Sets the indicator image for the specified page.
+    description: |
+        Override the indicator image for a specific page. Symbol images are recommended.
+        Use <Titanium.UI.iOS.systemImage> to get system-supplied symbol images.
+    parameters:
+      - name: image
+        summary: The image for the indicator. Resets to the default if image is null.
+        type: [String, Titanium.Blob]
```

Comment:
What sort of Strings are allowed? A file path to an image? (again an example in the docs may be helpful here)

Reply:
Updated doc.